io.github.am2rcommunitydevelopers.AM2RLauncher.yaml

```diff
@@ -0,0 +1,134 @@
+app-id: io.github.am2rcommunitydevelopers.AM2RLauncher
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+  - org.freedesktop.Sdk.Extension.openjdk11
+  - org.gnome.Sdk.Compat.i386
+
+finish-args:
+# TODO: crashes with wayland on filepicker and font being whacky. Likely Eto bug, test and report there.
+- --socket=x11
+- --share=ipc
+
+# Network needed for updates, pulseaudio for sound
+- --share=network
+- --socket=pulseaudio
+
+# Home for getting the 1.1 zip, config because of profiles having their saves there, apktool for apk creation
+- --filesystem=home:ro
+- --filesystem=~/.config/
```

Comment:
why not `--persist` ?

Reply:
Done locally, will commit once xdelta3 gets solved